planet4-gutenberg-blocks.php

```diff
@@ -440,6 +440,23 @@ function ( $std_class_object ) {
 	],
 ];
 
+add_filter(
+	'render_block_core/query-pagination',
+	function ( $content ) {
+		// Check if prev and next buttons are disabled.
+		if (
+			str_contains( $content, 'wp-block-query-pagination-previous disabled' )
+			&& str_contains( $content, 'wp-block-query-pagination-next disabled' )
+		) {
+			return '<div class="wp-block-query-pagination is-empty"></div>';
```

Comment:
Maybe instead of creating this new `is-empty` class, you could use Boostrap's `d-none` that does what you want? Then [the other PR](https://github.com/greenpeace/planet4-master-theme/pull/1830) would no longer be necessary 🙂 
Or actually maybe you could just return `null` since the pagination is not needed at all in this case 🤔

Reply:
Yes! that make sense, good point! we can return an empty string too. 
But the main reason of why I do that, is because we can style directly through CSS and keep the element. A same situation happens with the Next and Prev button. We've added the `disabled` instead of don't return them.
I'll ask the design team of what do they prefer.

Reply:
I think that was in case you are on the first or last page, the design team wanted the prev/next controls to still be shown but disabled. This is different since there's only one page 🤔